src/offset_of.rs

```diff
@@ -180,6 +217,27 @@ mod tests {
         assert_eq!(f_ptr as usize + 8, raw_field!(f_ptr, Foo, c) as usize);
     }
 
+    #[cfg(tuple_ty)]
+    #[test]
+    fn test_raw_field_tuple() {
+        let t = (0u32, 0u8, false);
+        let t_ptr = &t as *const _;
+        let t_addr = t_ptr as usize;
+
+        assert_eq!(
+            offset_of_tuple!((u32, u8, bool), 0),
+            raw_field_tuple!(t_ptr, (u32, u8, bool), 0) as usize - t_addr
+        );
```

Comment:
This is not really testing anything, as `offset_of_tuple!` uses `raw_field_tuple!` under the hood.

A better test might be something where you add the raw field offset to `t_addr` and compare that with a pointer directly to the corresponding field.